app/styles/templates/project/thankyou.scss

```diff
@@ -1,14 +1,27 @@
 .thankyou-container {
-  .icon, .header, .text, .project-link{
+  .icon, .header, .text, .project-link {
     width: 100%;
     text-align: center;
   }
 
-  .icon img {
-    display: inline-block;
+  .header, .text {
+    margin-bottom: 20px;
+  }
+
+  .header {
+    font-size: 1.5em;
+    margin-top: 0px;
+  }
+
+  .icon {
+    img {
+      display: inline-block;
+    }
+    margin-bottom: 0px;
   }
 
   .text {
-    margin-bottom: 18px;
+    color: $light-text;
+    font-size: 14px;
```

Comment:
on #703 I added "font-size: $body-font-size-normal" to the html element, that sets the baseline across the whole app to 14px so there's no need to set it here.

For future reference, we have four font sizes defined on _fonts.scss: 12px (small), 14px (normal), 16px (large) and 18px (x-large).